### PR TITLE
Nav bar fix

### PR DESCRIPTION
**Pull request by limcaaarl**
## Description

- Fixed the hamburger issue by replacing menubar item with menu component
- Downloaded poppins font locally

### CODE REVIEW

**Review by samuelim01 — APPROVED** [review]
LGTM, shall we redirect this PR to main directly though?

**Comment by limcaaarl** [comment]
> LGTM, shall we redirect this PR to main directly though?

This branch was from merge-all-in and there's some merge conflicts when merging with main. I think I will wait for the match-service merge first before merging this one, in case I make some merge conflict mistakes.

**Pull request merged**
